src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -33,10 +33,11 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 				return (<ConnectionProfile>element).id;
 			} else if (element instanceof ConnectionProfileGroup) {
 				return (<ConnectionProfileGroup>element).id;
-			} else if (element.nodeTypeId === 'Table' || element.nodeTypeId === 'Column') {
+			} else if (element.nodeTypeId === 'Table' || element.nodeTypeId === 'Column' || element.nodeTypeId === 'View') {
```

Comment:
1. At this point we should make an array of supported node types. This is already starting to get long and having it in one place means we're less likely to miss updating one of the places in the future
2. We should really just do a quick pass through and pick out all the objects that make sense. But also this seems like the kind of thing where most of the nodes are going to have the same fields and so I'd ask why we wouldn't just enable this for all nodes as long as they have the required fields. Sure, it might end up being that some nodes are included that don't necessarily make sense but better having too many than having to keep coming back and adding types a couple at a time.

Reply:
ok I'll have a chat with Alan about which objects to choose